dispatcher_plugin_nb2workflow/queries.py

```diff
@@ -10,35 +10,38 @@
 from .dataserver_dispatcher import NB2WDataDispatcher
 from cdci_data_analysis.analysis.ontology import Ontology
 import os
-from functools import lru_cache
+from functools import lru_cache, wraps
 from json import dumps
+from copy import deepcopy
 
 class HashableDict(dict):
     def __hash__(self):
-        return hash(dumps(self))
+        return hash(dumps(self, sort_keys=True))
 
 def with_hashable_dict(func):
+    @wraps(func)
     def wrapper(backend_param_dict, ontology_path):
         return func(HashableDict(backend_param_dict), ontology_path)
     return wrapper
 
-
 @with_hashable_dict
 @lru_cache
 def construct_parameter_lists(backend_param_dict, ontology_path):
     src_query_pars_uris = { "http://odahub.io/ontology#PointOfInterestRA": "RA",
                             "http://odahub.io/ontology#PointOfInterestDEC": "DEC",
                             "http://odahub.io/ontology#StartTime": "T1",
                             "http://odahub.io/ontology#EndTime": "T2",
-                            "http://odahub.io/ontology#AstrophysicalObject": "src_name"}
+                            "http://odahub.io/ontology#AstrophysicalObject": "src_name",
+                            "ThisNameShouldNotExist": "token"
```

Comment:
This is to fix https://github.com/oda-hub/dispatcher-plugin-nb2workflow/issues/87
Like this, if there is a parameter named "token" in the notebook, it will be renamed to "token_rename" in dispatcher.

Otherwise, this parameter will mask the special parameter "token" leading to authorization problems.